src/app/tests/TestReportScheduler.cpp

```diff
@@ -352,28 +350,29 @@ class TestReportScheduler
 
         NL_TEST_ASSERT(aSuite, CHIP_NO_ERROR == readHandler1->SetMaxReportingInterval(2));
         NL_TEST_ASSERT(aSuite, CHIP_NO_ERROR == readHandler1->SetMinReportingIntervalForTests(1));
-        ReadHandlerNode * node = sScheduler.FindReadHandlerNode(readHandler1);
-        node->SetIntervalTimeStamps(readHandler1);
-        readHandler1->MoveToState(ReadHandler::HandlerState::GeneratingReports);
+        sScheduler.OnReadHandlerSubscribed(readHandler1);
+        readHandler1->ClearStateFlag(ReadHandler::ReadHandlerFlags::PrimingReports);
```

Comment:
This does not match the order of those calls in the actual code, right?  We should have the same order here as there.